sklearn/gaussian_process/kernels.py

```diff
@@ -79,6 +79,42 @@ class Hyperparameter(namedtuple('Hyperparameter',
         changed during hyperparameter tuning. If None is passed, the "fixed" is
         derived based on the given bounds.
 
+    Examples
+    --------
+    >>> from sklearn.gaussian_process.kernels import ConstantKernel, RBF
+    >>> from sklearn.datasets import make_friedman2
+    >>> from sklearn.gaussian_process import GaussianProcessRegressor
+    >>> from sklearn.gaussian_process.kernels import Hyperparameter
+    >>> X, y = make_friedman2(n_samples=50, noise=0, random_state=0)
+    >>> kernel = ConstantKernel(constant_value=1.0,
+    ...    constant_value_bounds=(0.0, 10.0)) * RBF(length_scale=0.5,
+    ...    length_scale_bounds=(0.0, 10.0)) + RBF(length_scale=2.0,
+    ...    length_scale_bounds=(0.0, 10.0))
+    >>> for hyperparameter in kernel.hyperparameters: print(hyperparameter)
+    Hyperparameter(name='k1__k1__constant_value', value_type='numeric',
+    bounds=array([[ 0., 10.]]), n_elements=1, fixed=False)
+    Hyperparameter(name='k1__k2__length_scale', value_type='numeric',
+    bounds=array([[ 0., 10.]]), n_elements=1, fixed=False)
+    Hyperparameter(name='k2__length_scale', value_type='numeric',
+    bounds=array([[ 0., 10.]]), n_elements=1, fixed=False)
+    >>> params = kernel.get_params()
+    >>> for key in sorted(params): print("%s : %s" % (key, params[key]))
+    k1 : 1**2 * RBF(length_scale=0.5)
+    k1__k1 : 1**2
+    k1__k1__constant_value : 1.0
+    k1__k1__constant_value_bounds : (0.0, 10.0)
+    k1__k2 : RBF(length_scale=0.5)
+    k1__k2__length_scale : 0.5
+    k1__k2__length_scale_bounds : (0.0, 10.0)
+    k2 : RBF(length_scale=2)
+    k2__length_scale : 2.0
+    k2__length_scale_bounds : (0.0, 10.0)
+    >>> gpr = GaussianProcessRegressor(kernel=kernel, alpha=5,
+    ...         random_state=0).fit(X, y)
+    >>> gpr.score(X, y)
+    0.7135961476859551
```

Comment:
```suggestion
    0.7135...
```

Reply:
Yes, you're right. The example is too long for the section. I will make this shorter and more aligned with the other examples in this section.